ocs_ci/utility/logging/utils.py

```diff
@@ -0,0 +1,86 @@
+from ocs_ci.ocs.resources.csv import CSV
+from ocs_ci.utility.utils import run_cmd, clone_repo
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import SyncImagesUnsuccessfulException
+
+import os
+import stat
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_scripts_from_repo():
+    """
+    The function clones the repo quay-pull-images
+    """
+
+    url = 'https://github.com/red-hat-storage/quay-pull-images'
+    location = constants.QUAY_PULL_IMAGES_DIR
+    clone_repo(url, location, branch='master', to_checkout=None)
+
+
+def pull_packages_from_quay():
+    """
```

Comment:
please have better comments, where it shows us what kind of exceptions it could raise, i see you have SyncImagesUnsuccessfulException so far